docs/celery.rst

```diff
@@ -66,13 +64,12 @@ Configuring and Running
 -----------------------
 
 We set some reasonable defaults for Celery in ``settings.py``. These can be
-overriden either in ``settings_local.py`` or via the command line when running
+overriden either in your ``.env`` file or via the command line when running
 ``manage.py celeryd``.
 
-In ``settings_local.py`` you should set at least this, if you want to use
-Celery::
+In your ``.env`` file you should set at least this, if you want to use Celery::
 
-    CELERY_ALWAYS_EAGER = False
+    CELERY_ALWAYS_EAGER = false
 
 This defaults to ``True``, which causes all task processing to be done online.
```

Comment:
Wait, that's correct, CELERY_ALWAYS_EAGER is enabled in local development. The text is a bit odd, I'll fix.